### PR TITLE
Add Activity Create and Delete Command in Developer Guide

### PR DESCRIPTION
**Pull request by ivanaitzliddat**
Add relevant details in developer guide regarding activity create and delete command

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -320,7 +320,62 @@ The general workflow of the `session /summary` command is as follows:
...
+<br>
+The general workflow of the `activity /create` command is as follows:
+1. The user input provided is passed to `SplitLah`.
+2. `SplitLah` then parses the input by using methods in the `Parser` class to obtain a `ActivityCreateCommand` object.
+3. `ActivityCreateCommand#run` method is then invoked to run the `activity /create` command.
+4. Once the command runs, `ActivityCreateCommand#run` method checks if there are duplicate names in the involved list. 
+   - If there are duplicate names in the involved list, a message indicating that there are duplicates is printed using `TextUi#printlnMessage`
+     and control is given back to `SplitLah`.
+5. The `ActivityCreateCommand` object updates the cost and cost list by invoking the `ActivityCreateCommand#updateCostAndCostList` method.
+6. The`ActivityDeleteCommand#run` method invokes the `Manager#getProfile` method to retrieve the `Profile` object which stores the list of sessions.
```
Comment (warrencxw) [inline docs/DeveloperGuide.md:340]: Did you forget something here?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -320,7 +320,62 @@ The general workflow of the `session /summary` command is as follows:
...
+7. Then, `Profile#getSession` method is called to retrieve the `Session` object which the activity that the user wishes to delete is stored in.
+    - If the session does not exist, a message indicating that there is no such session is printed using `TextUi#printlnMessage` and control is given back to `SplitLah`.
+    - Else, the `Session` object that the activity is stored in is returned.
+    - This process is omitted in the sequence diagram for the sake of brevity.
+8. Other getter methods are then called to obtain the necessary parameters used to instantiate an Activity object. These getter methods are omitted in the sequence diagram.
+9. The `ActivityCreateCommand` object then adds the respective costs to each `Person` object involved in the activity using the `ActivityCreateCommand#addAllActivityCost` method.
+10. Using the updated details as parameters, the `ActivityCreateCommand` object instantiates an `Activity` object to represent the new activity.
+11. The `Session#addActivity` method is called to add the new `Activity` object into the list of activities.
```
Comment (warrencxw) [inline docs/DeveloperGuide.md:348]: Consider adding a 'then' into the sentence to link the flow from point 10 to point 11.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -320,7 +320,62 @@ The general workflow of the `session /summary` command is as follows:
...
+4. Once the command runs, `ActivityDeleteCommand#run` method will invoke `Manager#getProfile` method to retrieve the `Profile` object which stores the list of sessions.
+5. The `Profile#getSession` method is called to retrieve the `Session` object which the activity that the user wishes to delete is stored in.
+    - If the session does not exist, a message indicating that there is no such session is printed using `TextUi#printlnMessage` and control is given back to `SplitLah`.
+    - Else, the `Session` object that the activity is stored in is returned.
+6. Once the `Session` object is retrieved, the `Session#removeActivity()` method is invoked to remove the `Activity` object from the list of activities stored.
+   - If the activity does not exist, a message indicating that there is no such activity is printed using `TextUi#printlnMessage` and control is given back to `SplitLah`.
+   - Else, the `Activity` object is removed from the list of activities.
+7. The `ActivityCreateCommand` object updates the cost and cost list by invoking the `ActivityCreateCommand#updateCostAndCostList` method.
```
Comment (warrencxw) [inline docs/DeveloperGuide.md:374]: I think you forgot something here.

**Review by warrencxw — CHANGES_REQUESTED** [review]
Consider correcting the few things that I have pointed out in the comments. 

Additionally, for the ActivityCreateCommand sequence diagram, hasNameDuplicates should be placed outside of the Opt frame. Otherwise, you are implying that the method only runs if there are duplicates.

**Review by warrencxw — APPROVED** [review]
Looks good to merge.

**Review by froststein — APPROVED** [review]
Looks good to merge

**Pull request merged**